### ISSUE TITLE
[Windows] UIA Regression with Windows Narrator

### ISSUE THREAD

**Issue by yaakovschectman** [flutter/flutter#121907]
Hitting TAB to move the focus with UIA when using Windows Narrator does not read out the newly focused item, though NVDA does.
